### PR TITLE
CPP-242 - Added string functions for the CassInet type

### PR DESCRIPTION
**Pull request by mpenick**

### CODE REVIEW

File: src/types.cpp [hunk 1]
```diff
@@ -56,6 +58,32 @@ CassInet cass_inet_init_v6(const cass_uint8_t* address) {
...
 
+void cass_inet_string(CassInet inet, char* output) {
+  uv_inet_ntop(inet.address_length == CASS_INET_V4_LENGTH ? AF_INET : AF_INET6,
+               inet.address,
+               output, CASS_INET_STRING_LENGTH);
+}
+
+CassError cass_inet_from_string(const char* str, CassInet* output) {
+#if UV_VERSION_MAJOR == 0
+  if (uv_inet_pton(AF_INET, str, output->address).code == UV_OK) {
```
Comment (avalanche123) [inline src/types.cpp]: this makes it impossible to create an ipv6 inet value on a system that doesn't have ipv6 support right?

Comment (mpenick) [inline src/types.cpp]: @avalanche123 libuv implements custom implementations of those functions. https://github.com/libuv/libuv/blob/v1.x/src/inet.c#L161-L313

File: include/cassandra.h [hunk 2]
```diff
@@ -131,6 +131,11 @@ typedef struct CassString_ {
 #define CASS_INET_V6_LENGTH 16
 
 /**
+ * The size of an inet string including a null terminator.
+ */
+#define CASS_INET_STRING_LENGTH 46
```
Comment (aholmberg) [inline include/cassandra.h]: No change requested, but just an aside:
I kind of liked the way they defined this size with no 'magic numbers' in the old branch of libuv:
https://github.com/libuv/libuv/blob/v0.10/src/inet.c#L81

It's removed in 1.0, though.

**Comment by aholmberg** [comment]
+1 on changes

File: include/cassandra.h (hunk 2 above)
Comment (mpenick) [inline include/cassandra.h]: That's pretty cool. I'm going to keep the magic number because it follows the pattern of the other magic constants.

**Comment by mpenick** [comment]
Final commit: 39aa618b2a34842e02af8ca3510ae9ab427708f3

**Pull request closed without merging**